### DISCUSSION TITLE
Conditional Probability

### DISCUSSION THREAD

**Discussion by PARODBE** [networkx/networkx#5027] [Q&A]
Hi Evryone!,

I am new using networkx and I have a question. I would like calculate conditional probability for a new entry, how It would be possible? For example in the typical problem:

![image](https://user-images.githubusercontent.com/62376157/130125519-baa325cb-688c-437e-9d31-91b673d07b83.png)

How I could introduce some characteristics and obtaing something like this?

![image](https://user-images.githubusercontent.com/62376157/130125680-1aee8a31-3072-4cde-b56c-64ffb88b9a27.png)

You can imagine for example If i have a bronquitis, a rayX positive and I don't know whatever thing, what it would be the probability of have dysnea?

I know that is possible enter the weights to hand but I would like calculate this conditional probability when you do a manual process and also when the networkx learns the relationship between variables by itself.

Kind regards,
Pablo

**Comment by vpozdnyakov** [comment]
Could you explain what kind of distributions you want to model in terms of graphs (nodes, edges, etc.)?
